aip/general/0140.md

```diff
@@ -88,6 +88,31 @@ For uniformity, field names that contain both a noun and an adjective
 - `collected_items` (**not** `items_collected`)
 - `imported_objects` (**not** `objects_imported`)
 
+### Verbs
+
+Field names **must not** be expressed to reflect an intent or action.
+That is, they **must not** be expressed as verbs. Rather, the field name
+should express the condition of that aspect of the resource as defined
+by the value of the field in question.
+
+When performing a mutation, the field value is set to the desired
```

Comment:
this feels like something that belongs in "rationale" to me, but it's not required.